site/gatsby-site/i18n/locales/es/popovers.json

```diff
@@ -0,0 +1,54 @@
+    },
+    "incident_date": {
+        "title": "¿Cuándo ocurrió el incidente por primera vez?",
+        "text": "Muchos incidentes de IA no son eventos discretos (es decir, ocurren durante un período de tiempo). Además, a menudo es difícil saber cuándo ocurrió el incidente por primera vez cuando el primer informe público sigue un período de tiempo después de la primera ocurrencia. En este campo, debes ingresar la fecha del primer informe del incidente o una suposición fundamentada de cuándo ocurrió probablemente el incidente por primera vez. Por ejemplo, si el incidente pertenece a una característica del motor de búsqueda que se lanzó en enero y el primer el informe de incidente público es en febrero, debes asignar una fecha de incidente en enero"
+    },
+    "date_published": {
+        "title": "¿Cuándo estuvo disponible el informe?",
+        "text": "Los informes suelen tener una fecha de publicación en la URL, pero incluso cuando no la tiene, puede visitar el <linkto to=\"https://archive.org/web/\"> Wayback Machine</linkto> para encontrar cuándo se indexó por primera vez la URL del informe.texto:"
+    },
+    "date_downloaded": {
+        "title": "¿Cuándo copiaste el contenido del informe en este formulario?",
+        "text": "Las organizaciones de noticias suelen actualizar los informes a lo largo del tiempo. Esta fecha ayuda a determinar si el contenido podría estar obsoleto. Por lo general, colocará el título de hoy: la fecha de aquí"
+    },
+    "url": {
+        "title": "Enlace al informe disponible públicamente",
+        "text": "Esta dirección estará vinculada dentro de muchas interfaces de usuario de bases de datos de incidentes"
+    },
+    "image_url": {
+        "title": "Una imagen utilizada para obtener una vista previa del informe del incidente",
+        "text": "Esta debería ser una URL para una imagen que se indexará y se mostrará junto al informe. La mayoría de las publicaciones ahora tienen imágenes de titulares de las que puede obtener una URL haciendo clic con el botón derecho. También puede vincular imágenes de figuras aquí. "
+    },
+    "incident_id": {
+        "title": "¿Es este un incidente existente?",
+        "text": "Muy a menudo, los informes se escriben sobre incidentes que ya están en la base de datos de incidentes. Antes de enviarlos, debe verificar si el incidente ya está en la base de datos visitando la aplicación \"Discover\" y buscando incidentes similares. naturaleza al informe de incidente que está enviando ahora. Si existe un incidente para su informe, pero el informe nunca se ha enviado, ingrese aquí el número de identificador de incidente correspondiente. Si el informe de incidente ya existe, no lo vuelva a enviar"
```

Comment:
Change `buscando incidentes similares. naturaleza` to `buscando incidentes de similar naturaleza`